x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -363,6 +363,13 @@ export class EndpointDocGenerator {
       '@timestamp': ts,
       event: {
         created: ts,
+        id: this.seededUUIDv4(),
+        kind: 'metric',
+        category: 'host',
```

Comment:
`category` and `type` can be an array per the ecs spec. I believe the endpoint will actually send these values as an array of a single value for metadata.

My suggestion would be to mimic that and do `category: ['host']` `type: ['info']`

Reply:
updated